Core/HLE/sceIo.cpp

```diff
@@ -21,6 +21,7 @@
 #include <memory>
 
 #include "Common/Thread/ThreadUtil.h"
+#include "Common/TimeUtil.h"
```

Comment:
This header is not needed after clean up too.

Reply:
Ah, oops.